.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md
```

Comment:
-  [ Optional] The `test.md` file is a requirement for a previous project. Kindly take it out as it's not needed in this project